File: OMKARAUTOAPP/src/app/models/vehicleinstschedulelistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Vehicleinstschedulemodel } from "./vehicleinstschedulemodel";

export class Vehicleinstschedulelistmodel {
    vehicleInstSchedulelist: Vehicleinstschedulemodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: OMKARAUTOAPI/FleetTrans/Models/VehicleInstScheduleListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Shared.Models;

namespace FleetTrans.Models
{
    public class VehicleInstScheduleListModel
    {
        public List<VehicleInstScheduleModel> VehicleInstSchedulelist { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
